File: vuebugtrackerproject.client/src/viewmodels/ProjectContainer.ts
import type ProjectPreviewViewModel from "./ProjectPreviewViewModel";

// Container for returning a set of projects from the server.
export default class ProjectController{
  // The number of projects the user can see.
  totalProjects: number = 0;
  // The projects the user can view.
  projects: ProjectPreviewViewModel[] = [];
}

File: VueBugTrackerProject.Classes/ViewModels/ProjectContainer.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace VueBugTrackerProject.Classes
{
    /// <summary>
    /// Container for returning a set of projects from the server.
    /// </summary>
    public class ProjectContainer
    {
        /// <summary>
        /// The number of projects the user can see.
        /// </summary>
        public int TotalProjects { get; set; }

        /// <summary>
        /// The projects the user can view.
        /// </summary>
        public List<ProjectPreviewViewModel> Projects { get; set; } = new List<ProjectPreviewViewModel>();
    }
}
